src/vs/workbench/common/editor/resourceEditorInput.ts

```diff
@@ -116,10 +120,14 @@ export abstract class AbstractResourceEditorInput extends EditorInput implements
 		}
 	}
 
+	private get descriptionResource(): URI {
```

Comment:
Whats the idea here, why this change?

Reply:
We very often show the description (as hover or label description) of the editor input when we show the name of the editor input. If the name is using a custom label then we should show the original file name in the description. 
When there is no custom label, then it's easy to know which file is being represented, it's just `description/name`, but when there is a custom label, this is not the case anymore, so we should make sure the description contains all the information.

Reply:
@benibenj I think I would limit this to hover only, thus `getTitle` but not description. Otherwise, wouldn't the description often repeat what the name already shows?

Reply:
there is a good chance it will repeat, but it won't always be the case. Some users have `${dirname}.${extname}` for example. I'll remove it for the time being and check what users say.

Reply:
Thanks!